.github/workflows/workflow-run-forge.yaml

```diff
@@ -128,6 +128,7 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_DOCKER_ARTIFACT_REPO: ${{ secrets.AWS_DOCKER_ARTIFACT_REPO }}
           GIT_CREDENTIALS: ${{ secrets.GIT_CREDENTIALS }}
+          GCP_AUTH_DURATION: ${{ inputs.FORGE_RUNNER_DURATION_SECS * 2 }}
```

Comment:
@perryjrandall I think this broke the run:

 Invalid workflow file: .github/workflows/forge-stable.yaml#L103
The workflow is not valid. In .github/workflows/forge-stable.yaml (Line: 103, Col: 11): Error from called workflow aptos-labs/aptos-core/.github/workflows/workflow-run-forge.yaml@main (Line: 131, Col: 30): Unexpected symbol: '*'. Located at position 35 within expression: inputs.FORGE_RUNNER_DURATION_SECS * 2